web/packages/shared/components/UnifiedResources/FilterPanel.tsx

```diff
@@ -494,6 +522,152 @@ function ViewModeSwitch({
   );
 }
 
+export const IncludedResourcesSelector = ({
+  onChange,
+  includedResourceMode,
+}: {
+  onChange: (value: IncludedResourceMode) => void;
+  includedResourceMode: IncludedResourceMode;
+}) => {
+  const options: { value: IncludedResourceMode; label: string }[] = [
+    {
+      value: 'accessible',
+      label: 'Accessible',
+    },
+    {
+      value: 'requestable',
+      label: 'Requestable',
+    },
+  ];
+
+  const [availability, setAvailability] = useState<IncludedResourceMode[]>(
+    includedResourceMode === 'all'
+      ? ['requestable', 'accessible']
+      : [includedResourceMode]
+  );
```

Comment:
to be consistent, should we not select any checkbox for `all` initally? i think this is what we do for `types` filter. initially we show everything, and no types are selected

<img width="258" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/2a7161e3-da8b-4361-bf75-d9408a030089">

Reply:
sure. all/none selected are functionally the same so I can update it to be consistent